.gitignore

```diff
@@ -56,10 +56,6 @@ typings/
 .env.development
 .env.production
 
-# gatsby files
-.cache/
-public
```

Comment:
This is introducing issues where public folders aren't commited correctly

Reply:
Thank you for bringing this to our attention. I'll go ahead and take care of this since I was unaware that folder was in `.gitignore` (likely from the prior code).